dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.6/com.ibm.websphere.appserver.org.eclipse.persistence-2.6.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.6
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink; apiJar=false; location:=dev/api/third-party/, \
+  com.ibm.ws.javaee.persistence.2.1; location:=lib/
+kind=beta
+edition=core
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.persistence-1.0))", \
```

Comment:
`IBM-Provision-Capability` and `IBM-Install-Policy` are only needed for auto-features, so these clauses can be removed